deluge-pip.json

```diff
@@ -2,17 +2,17 @@
     "name": "deluge-pip",
     "plugin_schema": "2",
     "release": "12.2-RELEASE",
-    "artifact": "https://github.com/jsegaert/iocage-plugin-deluge-pip.git",
+    "artifact": "https://github.com/Richardgriff/iocage-plugin-deluge-pip.git",
```

Comment:
Why not open up a PR from your forked plugin repo to the official one instead of changing the artifact here?

Reply:
I'll be glad to accept that PR 😁

Reply:
Now switched back to jsegaert artifact. Pull requested under

https://github.com/jsegaert/iocage-plugin-deluge-pip/pull/2